### ISSUE TITLE
More flexibility in tweet intent

### ISSUE THREAD

**Issue by maelle** [rstudio/bookdown#936]
This is a feature request.

It'd be nice to be able to customize tweet intent URLs more for instance picking hashtags, via and related accounts https://developer.twitter.com/en/docs/twitter-for-websites/tweet-button/guides/web-intent

I acknowledge this is not a very critical feature. :-)

---

By filing an issue to this repo, I promise that

- [x] I have fully read the issue guide at https://yihui.org/issue/.
- [x] I have provided the necessary information about my issue.
    - If I'm asking a question, I have already asked it on Stack Overflow or RStudio Community, waited for at least 24 hours, and included a link to my question there.
    - If I'm filing a bug report, I have included a minimal, self-contained, and reproducible example, and have also included `xfun::session_info('bookdown')`. I have upgraded all my packages to their latest versions (e.g., R, RStudio, and R packages), and also tried the development version: `remotes::install_github('rstudio/bookdown')`.
    - If I have posted the same issue elsewhere, I have also mentioned it in this issue.
- [x] I have learned the Github Markdown syntax, and formatted my issue correctly.

I understand that my issue may be closed if I don't fulfill my promises.

**Comment by cderv** [comment]
The link opens a windows on twitter where you can add or delete some hastags. So you can customize what you are posting.

Do you want to be able to change the default from your yaml `sharing` field, is that it ?

**Comment by maelle** [comment]
yes, for instance I'd set "via" to my username or the org username.
